media-gfx/povray/povray-3.7.0.8.recipe

```diff
@@ -77,14 +75,25 @@ BUILD()
 {
 	cd unix
 	./prebuild.sh
+	autoupdate
 	cd ..
 
-	export CXXFLAGS="-std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
 	export LDFLAGS="-lbsd -lnetwork"
+	export CXXFLAGS="-std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
 
-	runConfigure ./configure \
-		COMPILED_BY="HaikuPorts" \
-		--with-boost-libdir=`finddir B_SYSTEM_LIB_DIRECTORY`
+	if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
```

Comment:
Please use a configureArgs variable with additional architecture parameters. runConfigure should appear one time.

Reply:
Fixed.

Reply:
The fix is not a fix.

Reply:
I simplified it. If you have an official solution, patches welcome...